### PR TITLE
Spring security

### PR DESCRIPTION
**Pull request by hstrodtman**

### CODE REVIEW

**Review by KBeas21 — APPROVED** [review]
1

**Review by MulesKC — COMMENTED** [review]
looks great!

**Review by MulesKC — APPROVED** [review]

File: build.gradle [hunk 1]
```diff
@@ -21,6 +21,7 @@ repositories {
 
 dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter-security'
+	implementation 'org.thymeleaf.extras:thymeleaf-extras-springsecurity5'
```
Comment (KBeas21) [inline build.gradle:24]: this bad fix

**Review by jamiander — APPROVED** [review]

**Review by KBeas21 — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 1 above)
Comment (KBeas21) [reply build.gradle:24]: pushed change

**Review by KBeas21 — APPROVED** [review]

**Review by scottpenneman — APPROVED** [review]
looks good!

File: src/main/java/org/launchcode/water_garden_tour/controllers/LoginController.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+@Controller
+public class LoginController {
+
+    @RequestMapping(value="/login", method = {RequestMethod.GET, RequestMethod.POST})
+    String login() {
```
Comment (KBeas21) [inline src/main/java/org/launchcode/water_garden_tour/controllers/LoginController.java:11]: this bad

**Review by KBeas21 — CHANGES_REQUESTED** [review]

File: src/main/java/org/launchcode/water_garden_tour/controllers/LoginController.java (hunk 2 above)
Comment (KBeas21) [reply src/main/java/org/launchcode/water_garden_tour/controllers/LoginController.java:11]: thi was not bad i'm dumb

**Review by KBeas21 — APPROVED** [review]

**Pull request merged**